test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -669,6 +671,50 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 			})
 		})
 
+		When("perPodPowerManagent enabled", func() {
+			It("should update kernel arguments and tuned accordingly", func() {
+				profile.Spec.WorkloadHints = &performancev2.WorkloadHints{
+					PerPodPowerManagement: pointer.BoolPtr(true),
+					HighPowerConsumption:  pointer.BoolPtr(false),
+					RealTime:              pointer.BoolPtr(true),
+				}
+				By("Patching the performance profile with workload hints")
+				workloadHints, err := json.Marshal(profile.Spec.WorkloadHints)
+				Expect(err).ToNot(HaveOccurred())
+
+				Expect(testclient.Client.Patch(context.TODO(), profile,
+					client.RawPatch(
+						types.JSONPatchType,
+						[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec/workloadHints", "value": %s }]`, workloadHints)),
+					),
+				)).ToNot(HaveOccurred())
```

Comment:
let's make sure to revert this patch if the test fails early - and yes, other tests may have the same issue, so I'd like to introduce this test already fixed and then fix the others/file a issue to fix the others separately

Reply:
If we do that in AfterEach, that's enough indeed.